content/download/releases/v7-2-5.md

```diff
@@ -3,7 +3,7 @@ title: "7.2.5"
 date: 2024-04-15
 extra:
     tag: "7.2.5"
-    artifact_source: https://d307a34p6mmcbn.cloudfront.net/releases/
+    artifact_source: https://download.valkey.io/releases/
```

Comment:
Nice.

Reply:
Now, I think it should always be this `https://download.valkey.io/releases/` so we can add it to the `config.toml`

Reply:
@roshkhatri Okay! I'm not going to block on this because it's just an optimization, but could you please make an issue so we don't forget to move `artifact_source` to a global setting?

Reply:
Sure, here I have created an issue for the same: https://github.com/valkey-io/valkey-io.github.io/issues/131